hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2262,3 +2265,219 @@ def hpat_pandas_series_argsort_impl(self, axis=0, kind='quicksort', order=None):
     return hpat_pandas_series_argsort_impl
 
 
+@overload_method(SeriesType, 'sort_values')
+def hpat_pandas_series_sort_values(self, axis=0, ascending=True, inplace=False, kind='quicksort', na_position='last'):
+    """
+    Pandas Series method :meth:`pandas.Series.sort_values` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sort_values1
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sort_values2
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sort_values_index1
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sort_values_noidx
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sort_values_idx
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sort_values_parallel1
```

Comment:
You could use pattern to represent many tests via one line in docstring:
`python -m hpat.runtests -k hpat.tests.test_series.TestSeries.test_series_sort_values*`